bin/node-lambda

```diff
@@ -21,7 +22,7 @@ var AWS_HANDLER = process.env.AWS_HANDLER || 'index.handler';
 var AWS_ROLE = process.env.AWS_ROLE_ARN || process.env.AWS_ROLE || 'missing';
 var AWS_MEMORY_SIZE = process.env.AWS_MEMORY_SIZE || 128;
 var AWS_TIMEOUT = process.env.AWS_TIMEOUT || 60;
-var AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || '';
+var AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || packageJson.description || '';
```

Comment:
I think it will auto fallback to '' if that is the case.

2016-08-01 20:59 GMT+08:00 Chase <notifications@github.com>:

> In bin/node-lambda
> <https://github.com/motdotla/node-lambda/pull/133#discussion_r72973230>:
>
> > @@ -21,7 +22,7 @@ var AWS_HANDLER = process.env.AWS_HANDLER || 'index.handler';
> >  var AWS_ROLE = process.env.AWS_ROLE_ARN || process.env.AWS_ROLE || 'missing';
> >  var AWS_MEMORY_SIZE = process.env.AWS_MEMORY_SIZE || 128;
> >  var AWS_TIMEOUT = process.env.AWS_TIMEOUT || 60;
> > -var AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || '';
> > +var AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || packageJson.description || '';
>
> description is not a required package.json field, so we might want to
> check if description is in packageJson before trying to load it
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/motdotla/node-lambda/pull/133/files/cd09fb8a638bfd078f5afaee8414d0bb09f4f110#r72973230>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKsHf-CXTOMwd_3jFUwbnlg_2ecZVc6ks5qbe2hgaJpZM4JY7kY>
> .
>



-- 
Ulion